Translation corrections (#160)
07th-mod/higurashi-console-arcs: Update/some_tips_087.txt

			NULL, " I don't want Sis to end up like our father.", Line_WaitForInput);
	ModPlayVoiceLS(4, 29, "ps3/s20/29/vtt_madoka3038", 256, TRUE);
	OutputLine(NULL, "努力の数だけ頑張った人が損をするのは、…絶対、間違ってます」",

> 			NULL, " People who work hard and have nothing to show for it... can't possibly be right.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " People who work hard having nothing to show for it...? That can't possibly be right.\"", Line_Normal);
```
